src/components/accordion/AccordionIcon.tsx

```diff
@@ -1,11 +1,18 @@
 import React, { FC } from 'react';
-import { useAccordionItemState } from '@chakra-ui/react';
+import { useAccordionItemState, useAccordionStyles } from '@chakra-ui/react';
 
-import { ChevronDownLargeIcon, ChevronUpLargeIcon } from '../icons';
+import { ChevronUpLargeIcon } from '../icons';
 
 const AccordionIcon: FC = () => {
   const { isOpen } = useAccordionItemState();
-  return isOpen ? <ChevronUpLargeIcon /> : <ChevronDownLargeIcon />;
+  const styles = useAccordionStyles();
```

Comment:
There are some with the icon size, we can revert this as it did not actually making a difference